internal/providers/pke/pkeworkflow/create_aws_worker_pool_activity.go

```diff
@@ -122,9 +122,9 @@ func (a *CreateWorkerPoolActivity) Execute(ctx context.Context, input CreateWork
 	}
 
 	stackInput := &cloudformation.CreateStackInput{
-		StackName:    aws.String(stackName),
-		TemplateBody: aws.String(template),
-		// ClientRequestToken: aws.String(string(activity.GetInfo(ctx).ActivityID)),
+		StackName:          aws.String(stackName),
+		TemplateBody:       aws.String(template),
+		ClientRequestToken: aws.String(activity.GetInfo(ctx).WorkflowExecution.ID),
```

Comment:
Note: I explicitly highlight this change, because this isn't just a client token request (CRT) value replacement, but also an introduction of using CRT in this context.
I assumed the original value was commented out because activity ID was not a reliable CRT source as it changed between activity retries.